print.php

```diff
@@ -144,502 +143,576 @@
     } catch(Error $e){
         echo false;
     } finally {
-       
+       // mysqli_stmt_close($connect);
```

Comment:
Yeah, I think this is only need if using prepared statements. I removed this on other pages as well.